org-ac.el

```diff
@@ -211,6 +211,12 @@
     (cache)
     (action . ac-pcmp/do-ac-action)))
 
+(defvar ac-source-org-ac-file
```

Comment:
I think the following is better.

```lisp
(defvar ac-source-org-ac-file
  '((init . (setq ac-filename-cache nil))
    (candidates . org-ac/file-candidate)
    (prefix . "\\[file:\\(.*\\)")
    (symbol . "f")
    (requires . 0)
    (action . ac-start)
    (limit . nil)))
```

* Probably `init` key is wrong. see `(ac-define-source filename ...` in auto-complete.el
* `prefix` key uses a first group in reguler-expression for `ac-prefix`
* We can start completion even if `ac-prefix` don't exist giving 0 to `requires` key